parameters/create_parameters.jl

```diff
@@ -109,22 +109,28 @@ function create_climaatmos_parameter_set(
     )
     SFP = typeof(surf_flux_params)
 
-    aliases = [
-        "microph_scaling",
-        "microph_scaling_dep_sub",
-        "microph_scaling_melt",
-        "Omega",
-        "planet_radius",
-    ]
-    pairs = CP.get_parameter_values!(toml_dict, aliases, "TurbulenceConvection")
+    aliases = string.(fieldnames(TCP.TurbulenceConvectionParameters))
+    pairs = CP.get_parameter_values!(toml_dict, aliases, "EDMF")
     pairs = override_climaatmos_defaults((; pairs...), overrides)
-
     tc_params = TCP.TurbulenceConvectionParameters{FTD, MP, SFP}(;
         pairs...,
         microphys_params,
         surf_flux_params,
     )
 
+    config_param_names = [
+        "sgs",
+        "quadrature_type",
+        "quadrature_order",
+        "precip_fraction_model",
+        "prescribed_precip_frac",
+        "precip_fraction_limiter",
+        "thermo_covariance_model",
+        "diagnostic_covar_limiter"
```

Comment:
The `quadrature_order`, `prescribed_precip_frac`, `precip_fraction_limiter` and `diagnostic_covar_limiter` are all numbers. So they could still be part of the `param_set` and not `config_params`?

Reply:
They seemed to only get used for constructing the data structs, and I thought that we should use the `config_params `for model construction, but I can move them back into the `TurbulenceConvectionParameters` if that makes more sense.

Reply:
Either way works for me. I'm fine with them being config params too if there is a good software engineering reason for it.